documentation/staging/content/managing-domains/domain-lifecycle/scaling.md

```diff
@@ -16,36 +16,80 @@ WebLogic Server supports two types of clustering configurations, configured and
+
+##### kubectl scale command
+An individual WebLogic cluster can be scaled directly using the `kubectl scale` command. For example, the following command sets .spec.replicas of the Cluster resource cluster-1 to 5:
 ```shell
-$ kubectl edit domain domain1 -n [namespace]
+$ kubectl scale --replicas=5 clusters/cluster-1
+  clusters "cluster-1" scaled
+ 
+$ kubectl get clusters cluster-1 -o jsonpath='{.spec.replicas}'
+  5
+```
+
+##### kubectl patch command
+An individual WebLogic cluster can also be scaled directly using the `kubectl patch` command with your patch object inline. For example, the following command sets .spec.replicas of the Cluster resource `cluster-1` to 3:
+```shell
+$ kubectl patch cluster cluster-1 --type=merge -p '{"spec":{"replicas":3}}'
+  cluster.weblogic.oracle/cluster-1 patched
+ 
+$ kubectl get clusters cluster-1 -o jsonpath='{.spec.replicas}'
+  3
+```
+
+#### On-demand, updating the Cluster or Domain directly
+On-demand scaling (the ability to scale a cluster manually) of WebLogic clusters can be done by directly updating the `.spec.replicas` field of the Cluster or Domain resources.
+##### Updating the Cluster directly
+To scale an individual WebLogic cluster directly, one can simply edit the `replicas` field of its associated Cluster resource. This can be done by using `kubectl`. More specifically, you can modify the Cluster directly by using the `kubectl edit` command.  For example:
+```shell
+$ kubectl edit cluster cluster-1 -n [namespace]
 ```
-Here we are editing a Domain named `domain1`. The `kubectl edit` command will open the Domain definition in an editor and allow you to modify the `replicas` value directly. Once committed, the operator will be notified of the change and will immediately attempt to scale the corresponding cluster by reconciling the number of running pods/Managed Server instances with the `replicas` value specification.
+Here we are editing a Cluster named `cluster-1`. The `kubectl edit` command will open the Cluster definition in an editor and allow you to modify the `replicas` value directly. Once committed, the operator will be notified of the change and will immediately attempt to scale the corresponding cluster by reconciling the number of running pods/Managed Server instances with the `replicas` value specification.
 ```yaml
 spec:
   ...
-  clusters:
-    - clusterName: cluster-1
-      replicas: 1
+  clusterName: cluster-1
+  replicas: 1
   ...
 ```
-
-Alternatively, you can specify a default `replicas` value for all the clusters.  If you do this, then you don't need to list the cluster in the Domain (unless you want to customize another property of the cluster).
+##### Updating the Domain directly
+To scale a WebLogic cluster, that does not reference a Cluster resource or the Cluster resource does not have the `.spec.replicas` field specified, modify the value of the `replicas` field of a Domain by using the `kubectl edit` command.  
```

Comment:
Minor (no comma to start, extra that):
Minor: The note now looks a bit redundant.  How about removing the note and combining it with the introductory paragraph? Something like:

To scale every WebLogic cluster in a domain that does not have a corresponding Cluster resource, or that has a Cluster resource which does not specify a `cluster.spec.replicas` field, modify the `domain.spec.replicas` using the `kubectl edit` command: